monthly-contributions/src/main/scala/com/gu/support/workers/lambdas/UpdateMembersDataAPI.scala

```diff
@@ -0,0 +1,27 @@
+package com.gu.support.workers.lambdas
+
+import com.gu.support.workers.model.monthlyContributions.state.UpdateMembersDataAPIState
+import com.typesafe.scalalogging.LazyLogging
+import com.amazonaws.services.lambda.runtime.Context
+import com.gu.config.Configuration
+import com.gu.emailservices.EmailService
+import com.gu.membersDataAPI.MembersDataService
+
+import scala.concurrent.ExecutionContext.Implicits.global
+import com.gu.support.workers.encoding.StateCodecs._
+
+import scala.concurrent.Future
+
+class UpdateMembersDataAPI(membersDataService: MembersDataService)
```

Comment:
Are you supporting test users in the UAT environment? I think this Lambda should inherit from ServicesHandler and take its service dependency from there in the same way that CreateSalesforceContact does, this can then take care of passing in a service with the correct configuration (UAT, PROD or DEV). See ServiceProvider for where this is all set up for the other services